core/snapshot/engine.go

```diff
@@ -218,6 +219,40 @@ func (e *Engine) ReloadConfig(cfg Config) {
 	e.Config = cfg
 }
 
+// List returns all snapshots available.
+func (e *Engine) ListMeta() ([]*tmtypes.Snapshot, error) {
+	e.avlLock.Lock()
+	defer e.avlLock.Unlock()
+
+	snapshots := make([]*tmtypes.Snapshot, 0, len(e.versions))
+	// TM list of snapshots is limited to the 10 most recent ones.
+	i := len(e.versions) - 11
```

Comment:
I know it's copied from the list() but why the -11?

Reply:
no idea, I would have done -10, but I guess there's some black magic reason for it to be -11?

Reply:
it must be because it's the smallest two-digit prime number (◔_◔)

Reply:
it must definitely be the reason.